src/passkey_child/passkey_child_common.c

```diff
@@ -82,6 +84,8 @@ parse_arguments(int argc, const char *argv[], struct passkey_data *data)
          _("Register a passkey for a user"), NULL },
         {"authenticate", 0, POPT_ARG_NONE, NULL, 'a',
          _("Authenticate a user with a passkey"), NULL },
+        {"format", 0, POPT_ARG_NONE, NULL, 'f',
```

Comment:
I think something which mentions 'PEM' like `--format-pem` might be better in case we want/have to add other conversions, e.g. from PEM to the pubkey format, in future.